src/parser/bison_parser.y

```diff
@@ -1007,6 +1008,20 @@ datetime_field:
     |   YEAR { $$ = kDatetimeYear; }
     ;
 
+datetime_field_plural:
+        SECONDS { $$ = kDatetimeSecond; }
+    |   MINUTES { $$ = kDatetimeMinute; }
+    |   HOURS { $$ = kDatetimeHour; }
+    |   DAYS { $$ = kDatetimeDay; }
+    |   MONTHS { $$ = kDatetimeMonth; }
+    |   YEARS { $$ = kDatetimeYear; }
+    ;
+
+duration_field:
+		datetime_field
+	|	datetime_field_plural
```

Comment:
Similar to the test coverage comment regarding the different unit strings, we should have at least a small interval test for each possible duration field.